include/libdnf/repo/repo_sack.hpp

```diff
@@ -146,6 +156,9 @@ class RepoSack : public sack::Sack<Repo> {
 
     void internalize_repos();
 
+    void update_and_load_repos_singlethread(const libdnf::repo::RepoQuery & repos);
```

Comment:
Probably a typo `singlethread` => `singlethreaded`, I would unify the naming with the second method...